id/team.html

```diff
@@ -139,47 +139,49 @@ <h2>Our Team</h2>
             <div class="row">
               <div class="col-12 text-center" data-aos="fade-down" >
                 <img src="./../assets/img/team-1.jpg" alt="img-team" class="img-fluid rounded-circle mb-3" style="width: 12.5rem;">
-                <p class="fw-bold mb-0">Co-Founder & Direktur Utama</p>
```

Comment:
This file is obsolete. Please use the updated one

Reply:
done, you can check it aagain :D